doc/cephfs/quota.rst

```diff
@@ -74,6 +74,12 @@ To set a quota::
   setfattr -n ceph.quota.max_bytes -v 100000000 /some/dir     # 100 MB
   setfattr -n ceph.quota.max_files -v 10000 /some/dir         # 10,000 files
 
+``ceph.quota.max_bytes`` can also be set using human readable format
```

Comment:
I suggest s/human readable/human-friendly units (KiB, MiB, GiB, Tib)//

I also request that you add a note about these being base-2 units, since 100M here does not equal 100M above.  It is unfortunate that drive vendors embraced weasely base-10 units at all, but we're stuck with them in the mix, so I like to be explicit so that the reader knows exactly how to interpret units.  Once we get up into TB/TiB, PiB/PB, the different can be significant.

Reply:
> I suggest s/human readable/human-friendly units (KiB, MiB, GiB, Tib)//

ACK

> I also request that you add a note about these being base-2 units, since 100M here does not equal 100M above. It is unfortunate that drive vendors embraced weasely base-10 units at all, but we're stuck with them in the mix, so I like to be explicit so that the reader knows exactly how to interpret units. Once we get up into TB/TiB, PiB/PB, the different can be significant.

For this I can just change `# 100MB` to `# 100MiB` (the value written beside the command) right or a note is more preferred option?

Reply:
Just changing is fine, I guess we don't have to call out the difference on *every* instance.

Reply:
Thanks @anthonyeleven !